Interni/StudioDiFattibilità/components/fattibilità-c7.tex

```diff
@@ -0,0 +1,77 @@
+\section{Capitolato C7}
+
+\subsection{Informazioni generali}
+\begin{itemize}
+\item \textbf{Nome:} \textit{SSD}: Soluzioni di sincronizzazione desktop;
+\item \textbf{Proponente:} ZEXTRAS;
+\item \textbf{Committenti:} \textit{Prof. \Tullio{}} e \textit{Prof. \Riccardo{}}.
+\end{itemize}
+
+\subsection{Descrizione del capitolato}
+Il capitolato richiede lo sviluppo di un algoritmo e di un front-end per sincronizzare file tra server e client appoggiandosi sul sevizio Zextras Drive.
+L'interfaccia deve essere multipiattaforma e il prodotto deve essere sviluppato aderendo al pattern \glo{MVC}.
+
+\subsection{Finalità del progetto}
+Il capitolato pone i 3 seguenti obiettivi:
+\begin{itemize}
+\item \textbf{Sviluppo di un algoritmo di sincronizzazione solido ed efficiente}: deve garantire la sincronizzazione ottimale tra cloud e desktop utilizzando il quantitativo minimale di risorse;
+\item \textbf{Sviluppo di un interfaccia multipiattaforma}: deve fornire l'interfaccia necessaria per interagire con i servizi messi a disposizione 
+dall'algoritmo e essere utilizzabile almeno sui tre sistemi operativi più popolari (Linux, MacOS, Windows);
+La soluzione sviluppata non deve dipendere da installazioni di framework di terze parti da parte dell'utente finale;
+\item \textbf{Integrazione con Zextras Drive}: sia l'algoritmo che l'interfaccia devono essere integrati con il servizio di gestione file Zextras Drive.
+\end{itemize}
+Oltre al raggiungimento degli obbiettivi precedentemente riportati le seguenti funzionalità dovrebbero essere aggiunte:
```

Comment:
Le funzionalità aggiuntive sono sbagliate e da rivedere (in caso cancellare e basta)